Cargo.lock

```diff
@@ -949,6 +949,7 @@ dependencies = [
  "anyhow",
  "bytes",
  "deno_ast",
+ "deno_core",
```

Comment:
I wonder if this will cause problems during publishing... I never seen crate depend on itself

Reply:
It's a dev dependency to turn the force-include feature on for testing.